cmd/stork/stork.go

```diff
@@ -476,6 +477,12 @@ func runStork(mgr manager.Manager, ctx context.Context, d volume.Driver, recorde
 		adminNamespace = c.String("migration-admin-namespace")
 	}
 
+	// Setting up the generic cache. Currently it has the map to determine the right plural for a crd kind installed in the cluster.
+	if err := genericcache.CreateGenericCache(); err != nil {
```

Comment:
I feel instead of CreateGenericCache() It should be CreateCrdCache() as this cache is very specific to crds

Reply:
Or if we name the package crdcache instead of genericcache, we can name the function CreateCache()

Reply:
My intention in naming this as genericCache is that if in future we will require some more things need to be cached, we can make it as part of genericCache structure and provide the necessary methods to the interface. I agree that it is solving only crdPlurals for now.

Reply:
Ok. Let's keep package name genericCache. But we can name the func CreateCache()

Reply:
Done